### PR TITLE
Convert iOS guide to tutorial

### PR DESCRIPTION
**Pull request by tbedford**
## Description

This PR just converts a guide into a tutorial. This is an iOS tutorial. This PR is just the conversion. The tutorial itself needs some additional work to improve it.

~NOTE: Was created before the i18n merge, so all these files have since moved.~ FIXED.

## Review 

Check that the guide is now converted to a tutorial.

### CODE REVIEW

**Comment by fabianrbz** [comment]
@tbedford is it possible to put the redirect in the other file so it gets picked up by the GH action? How do we decide which is the right file?
I'll try to update the action as soon as possible.

**Comment by tbedford** [comment]
@fabianrbz good idea. I will try that shortly.

**Comment by tbedford** [comment]
@fabianrbz Now seems to give the error: 

```
Specified redirect could not be found: /github/workspace/_documentation/client-sdk/tutorials/ios-in-app-messaging.md
```

But in local testing `/client-sdk/tutorials/ios-in-app-messaging` goes quite happily to `/client-sdk/tutorials/ios-in-app-messaging/introduction`.

**Comment by tbedford** [comment]
@fabianrbz I think I have repaired this PR enough to get it back to the original problem. :)

File: _tutorials/en/client-sdk/ios-in-app-messaging/ios-add-user.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (fabianrbz) [inline _tutorials/en/client-sdk/ios-in-app-messaging/ios-add-user.md]: @tbedford do you think we still need the `ios` prefix in the filenames now that they are under `ios-in-app-messaging` or we can just drop it?

Comment (tbedford) [reply _tutorials/en/client-sdk/ios-in-app-messaging/ios-add-user.md]: @fabianrbz was wondering about that. I think we can drop it. I will sort it later today.

Comment (tbedford) [reply _tutorials/en/client-sdk/ios-in-app-messaging/ios-add-user.md]: Fixed.

File: config/redirects.yml [hunk 2]
```diff
@@ -251,6 +251,8 @@
 "/vonage-business-cloud/vbc-apis/user-api/code-snippets": "/vonage-business-cloud/vbc-apis/user-api/code-snippets/retrieve-user"
 "/vonage-business-cloud/vbc-apis/getting-started": "/vonage-business-cloud/vbc-apis/getting-started/create-application"
 "/send-an-sms": "/messaging/sms/code-snippets/send-an-sms"
+"/client-sdk/in-app-messaging/guides/ios-in-app-messaging": "/client-sdk/tutorials/ios-in-app-messaging"
+"/en/client-sdk/in-app-messaging/guides/ios-in-app-messaging": "/client-sdk/tutorials/ios-in-app-messaging"
```
[the remark this answers is not in the document]
Comment (tbedford) [reply config/redirects.yml]: I can remove it. It will go back to a different error.

**Pull request merged**